app/src/main/java/pl/gittobefit/network/WorkoutFormsServices.java

```diff
@@ -148,6 +148,27 @@ public void onResponse(Call<Training> call, Response<Training> response) {
                 }
                 else
                 {
+                    if (response.code() == 409)
+                    {
+                        switch (response.headers().get("Cause"))
+                        {
+                            case "0 body parts":
+                                activity.showSnackbar(fragment.getResources().getString(R.string.noBodyParts));
+                                break;
+                        }
+                    }
+                    else if (response.code() == 417)
+                    {
+                        switch (response.headers().get("Cause"))
+                        {
+                            case "wrong exercises count":
+                                activity.showSnackbar(fragment.getResources().getString(R.string.wrongCombination));
+                                break;
+                            case "not enough exercises for [BICEPS, SHOULDERS]":
```

Comment:
tu bys musiał sprawdzac czy jest not enough exercises for i w informacji wyswietlic dla jakich czesci ciała